modules/catalog/src/main/java/org/apache/ignite/internal/catalog/commands/CatalogUtils.java

```diff
@@ -433,23 +433,15 @@ public static int defaultLength(ColumnType columnType, int precision) {
      * @throws CatalogValidationException If schema with given name is not exists.
      */
     public static CatalogSchemaDescriptor schemaOrThrow(Catalog catalog, String name) throws CatalogValidationException {
-        name = Objects.requireNonNull(name, "schemaName");
-
-        CatalogSchemaDescriptor schema = catalog.schema(name);
-
-        if (schema == null) {
-            throw new CatalogValidationException("Schema with name '{}' not found.", name);
-        }
-
-        return schema;
+        return schema(catalog, name, true);
```

Comment:
after adding `@Nullable` to base method, it would be nice to do something with warning